include/volk/volk_complex.h

```diff
@@ -17,55 +17,32 @@
  * - lv_conj - take the conjugate of the complex number
  */
 
-#ifdef __cplusplus
-
-#include <stdint.h>
-#include <complex>
-
-typedef std::complex<int8_t> lv_8sc_t;
-typedef std::complex<int16_t> lv_16sc_t;
-typedef std::complex<int32_t> lv_32sc_t;
-typedef std::complex<int64_t> lv_64sc_t;
-typedef std::complex<float> lv_32fc_t;
-typedef std::complex<double> lv_64fc_t;
-
-template <typename T>
-inline std::complex<T> lv_cmake(const T& r, const T& i)
-{
-    return std::complex<T>(r, i);
-}
-
-template <typename T>
-inline typename T::value_type lv_creal(const T& x)
-{
-    return x.real();
-}
+// #include <volk/volk_common.h>
```

Comment:
why not use this & then in the next (valid) line do `__VOLK_DECL_BEGIN` (and also `__VOLK_DECL_END` at the end)?

Reply:
It didn't work with `__VOLK_DECL_BEGIN` etc. previously. However, it seems like it does now.